career-development/ladder.md

```diff
@@ -17,114 +17,150 @@ Here are some examples of other career ladders:
 1. Heavily Team Oriented
 1. High Employee Retention
 1. High Quality Code Base
-1. Open Communication 
+1. Open Communication
 1. Pair-Programs
 
-## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled 
+## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled
 
 ### Level 1
-#### Simple 
+#### Impact
+- [ ] Contributes positively to the development of one or multiple apps.
+
+#### Simple
 - [ ] Is working on naming variables and learning to extract methods.
-	
+
 #### Test-driven
 - [ ] Checks that all tests pass before merging into a shared branch.
 - [ ] Can understand where test expectations are not met and fix some with the help of more senior developers.
 - [ ] Can describe the testing pyramid and concepts around what makes a good test.
 - [ ] Writes “good” test descriptions: uses nesting and clear contexts/assertions
-	
+
 #### Iterative
 - [ ] Understands red-green-refactor.
-	
+
 #### Communication
 - [ ] Updates Asana.
 - [ ] Comes prepared for meetings.
 - [ ] Uses time tracking tools and vacation notifications.
-	
+
 #### Respectful teammate
 - [ ] On time for meetings.
 - [ ] Cares about their work.
-	
+
 #### Technical skills
 - [ ] Knows about the three main sections of a Git project(working, staging, repository) as well as the most common commands(commit, pull, push)
 - [ ] Uses a common linter such as rubocop
 - [ ] Can write code in at least one language.
 - [ ] Broad knowledge of main CS + XP concepts.
 
 ### Level 2
-#### Simple 	
+
+#### Impact
+- [ ] Capable of providing on-call support for their area including systems that they are not familiar with.
+
+#### Simple
 - [ ] Creates simple method / functions that most static code analysers would accept.
 - [ ] Makes clear prs
+
 #### Test-driven
 - [ ] Starts to write unit tests before code, writes acceptance tests.
 - [ ] Can write “acceptance” tests for existing code.
 - [ ] Refactors test code: extracts methods, breaks up long tests
-- [ ] Uses evident data in tests. 
-- [ ] Understands the trade-off between test isolation and test speed. 
+- [ ] Uses evident data in tests.
+- [ ] Understands the trade-off between test isolation and test speed.
+
 #### Iterative
 - [ ] Delivers working code with each commit.
-#### Communication	
+
+#### Communication
 - [ ] Hosts retros.
 - [ ] Can compose a PR and provide feedback on them.
+
 #### Respectful teammate
 - [ ] Shows respect for client goals
 - [ ] Encourages team to stay on a sustainable pace.
+
 #### Technical skills
 - [ ] Understands what differentiates their favourite language with others. Knows what the language is useful for and when it is not the best option.
 - [ ] Understand where to place logic in their favourite framework.
 
 ### Level 3
-#### Simple 
-- [ ] -
+
+#### Impact
+- [ ] End-to-end responsibility on projects of increasing complexity.
```

Comment:
Let's flesh out end-to-end a bit more. Perhaps dev-ops to engineering or engineering-technical PMing as two ranges to work with?

Reply:
Good point. What about?
Level 3:
- From internal meetings to internal acceptance
- Both backend and frontend implementation
Level 4:
- From client (research) to client acceptance
- Backend and frontend implementation